packages/core/src/components/Dialog/Dialog.tsx

```diff
@@ -457,6 +457,13 @@ export default class Dialog extends PureComponent<DialogProps, DialogState> {
   }
 
   onContextMenu(e: React.MouseEvent) {
+    const isShown = this.isShown();
+    if (
+      (this.isShowTrigger(HideShowEvent.CONTEXT_MENU) && !isShown) ||
+      (this.isHideTrigger(HideShowEvent.CONTEXT_MENU) && isShown)
```

Comment:
Can it mess up existing usages? Because it was already before, no?

Reply:
only the hide was in use, I talked with meirav and she want it to chenge to the existing users